haskell-indexer-backend-ghc/tests/Language/Haskell/Indexer/Backend/Ghc/Test/BasicTestBase.hs

```diff
@@ -50,7 +50,11 @@ testRecursiveRef = assertXRefsFrom "basic/RecursiveRef.hs" $ do
         _ -> checking $ assertFailure "Usage count differs"
     -- Recursive fun with type signature should target using the
     -- polymorphic binding.
-    declAt (11,1) >>= singleUsage >>= includesPos (11,16)
+    declAt (11,1) >>= usages >>= \case
+        [u1, u2] -> do
+            includesPos (10,1) u1
```

Comment:
Thanks! This is due to the newly emitted `ref/doc` reference from the type signature - can add a Note here, later we might sort if we actually want these to be counted in 'usage'.